.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "lib/sdl/SDL2_image"]
 	path = lib/sdl/SDL2_image
 	url = https://github.com/libsdl-org/SDL_image.git
+[submodule "lib/usb/ohci_usbhostlib"]
+	path = lib/usb/ohci_usbhostlib
+	url = https://github.com/Ryzee119/ohci_usbhostlib.git
```

Comment:
- Where to discuss https://github.com/Ryzee119/ohci_usbhostlib/compare/master...xbox?
- What's https://github.com/Ryzee119/ohci_usbhostlib/compare/xbox...timing?
- What are your plans for https://github.com/Ryzee119/nxdk_ohci_test/tree/master?

Reply:
* I created a self PR, happy to discuss stack specifics there. https://github.com/Ryzee119/ohci_usbhostlib/pull/1
* Timing branch was a initial attempt to minimise/remove some of the blocking delays in the stack. Have not spent any time on this yet. afair the whole stack is non-block except during device connection/removal/suspend stuff
* Master branch is/was intended to be a clean copy of upsteam to track diff.

Reply:
> I created a self PR, happy to discuss stack specifics there. Ryzee119/ohci_usbhostlib#1

Please add this information to the PR description - I assume it could be more important than the discussion on this repository.

>> What are your plans for https://github.com/Ryzee119/nxdk_ohci_test/tree/master?
>
> Master branch is/was intended to be a clean copy of upsteam to track diff.

I should probably have made a separate comment, but I was asking about "nxdk_ohci_test" and not "ohci_usbhostlib"; trying to get an idea of the existing ecosystem.

Reply:
> What are your plans for https://github.com/Ryzee119/nxdk_ohci_test/tree/master?
I think it could make a good nxdk sample if/when i clean it up to it suitable quality. Probably something I will do in the near future.